src/modules/beamos/filesystem/root/etc/init.d/beamos_first_boot

```diff
@@ -36,7 +43,10 @@ do_start () {
     # enable the uart / hw serial port
     # Disabled for now - activated during image creation
     #sudo raspi-config nonint do_serial 2
-    /sbin/reboot
+    if $REBOOT; then
+      echo "$(timestamp) Reboot device to apply changes" >> $logfile
+      /sbin/reboot
+    fi
```

Comment:
dunno if we need to log that,
could just go with `$REBOOT && /sbin/reboot`

Reply:
With this we can see that we did a reboot